app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_create :update_like_counter
+
+  def update_like_counter
```

Comment:
- [ ] Here as well you can make the method private as well.

Reply:
Solved :+1: